pages/css/architecture.md

```diff
@@ -3,36 +3,42 @@ title: Architecture
 permalink: /css/architecture/
 sidenav: css
 ---
-# Architecture
+# Choosing a CSS methodology and architecture
```

Comment:
I think my strongest opinion when it comes to our CSS is that we're either writing a quick throwaway prototype or something we intend to hand over and have our partner maintain, and we need to figure that out and behave accordingly from the get-go. For the former, I don't care what you write or how you write it, or if you follow any methodology at all beyond the mad scientist voices in your head. For the latter, I don't care how smart you are, you can't make the deliverable smarter than their average developer.